handsontable.d.ts

```diff
@@ -522,7 +523,7 @@ declare namespace Handsontable {
       isHorizontal(): boolean;
       isVertical(): boolean;
       removeTable(): void;
-      setSettings(settings: object): void;
+      setSettings(settings: DefaultSettings): void;
```

Comment:
This is a great improvement! I don't see a test for it, though (or for getSettings)

Reply:
Done